frontend/src/components/AccountCard/AccountCard.tsx

```diff
@@ -1,15 +1,12 @@
 import React from 'react';
-import { Card, BankAccount, Account } from './../../models/accounts';
+import { Card, BankAccount, Account, DigitalAccount } from './../../models/accounts';
 import { AccountCardPresentational } from './AccountCardPresentational';
+import { AccountCardProps as Props } from './Props';
 import './AccountCard.css';
 
 
-interface Props {
-    account: Account
-}
-
 function determineIfCard(toBeDetermined: Account): toBeDetermined is Card {
-    if((toBeDetermined as Card).used != null){
+    if((toBeDetermined as Card).cut != null){
```

Comment:
Codacy found an issue: [Expected '!==' and instead saw '!='.](https://app.codacy.com/manual/lgruelas/finance/pullRequest?prid=5891730)